src/AppRun.c

```diff
@@ -33,41 +33,68 @@ THE SOFTWARE.
 #include <string.h>
 #include <errno.h>
 
+/** Macro to throw error and exit. */
```

Comment:
So we are not changing this ?

Reply:
Aw criminy, I could have sworn I changed that thing! Gnargh...